src/adapters/simple_interest_loan_adapter.ts

```diff
@@ -80,6 +80,10 @@ export const SimpleInterestAdapterErrors = {
         singleLineString`Terms contract parameters are invalid for the given debt order.
                          Principal token at address ${principalTokenAddress} does not
                          correspond to specified token with symbol ${symbol}`,
+    MISMATCHED_TERMS_CONTRACT: (termsContract: string) =>
+        singleLineString`Terms contract at address ${termsContract} is not
+                         a SimpleInterestTermsContract.  As such, this adapter will not
+                         interface with the terms contract as expected.`,
```

Comment:
Not a big deal, but there is actually a grammar inconsistency here - see the above error message does not end in a period.